Library/Formula/premake.rb

```diff
@@ -1,8 +1,8 @@
 class Premake < Formula
-  desc "Premake is a build script generator"
-  homepage "http://industriousone.com/premake"
-  url "https://downloads.sourceforge.net/project/premake/Premake/4.3/premake-4.3-src.zip"
-  sha256 "36536490f8928d8ecde135da80cd8b751ea5bebe50cabba5c0de49cd41cb2780"
+  desc "Write once, build anywhere Lua-based build system"
+  homepage "https://premake.github.io/"
+  url "https://downloads.sourceforge.net/project/premake/Premake/4.4/premake-4.4-beta5-src.zip"
```

Comment:
Looks like development on 4.x has basically stalled out in favor of 5.x, and no real 4.4 release has been made yet. That https://premake.github.io is a pretty thin landing page that they're still working on filling out.

https://github.com/premake/premake-4.x/wiki/Development_Roadmap

https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!searchin/premake-development/4.4/premake-development/i1uA1Wk6zYM/kbp9q4Awu70J

If you really need access to the 4.4 prereleases, you could add it as a `devel` option.